docs/personalization/api_reference/recommendation_api.md

```diff
@@ -88,6 +88,17 @@ For more information, see [Submodels]([[= user_doc =]]/personalization/recommend
     
     `GET https://reco.perso.ibexa.co/api/v2/00000/john.doe/landing_page.json?numrecs=50&outputtypeid=1&width-range=10:30&color=green`
 
+!!! note "Dynamic attribute submodels"
+
+    If dynamic attribute submodels are enabled, you only need to add submodel parameters to the request to trigger dynamic submodels in the upcoming model build.
+
+    Dynamic attribute submodels must be enabled by [[= product_name_base =]] Team.
+    To start using this functionality, contact support@ibexa.co.
+
+    When enabled, to build dynamic attribute submodels, you need to send the request, which includes `"ATTRIBUTE_NAME=VALUE"`, to the scenario with model, you want to use submodels for:
```

Comment:
```suggestion
    When enabled, to build dynamic attribute submodels, you need to send the request, which includes `"ATTRIBUTE_NAME=VALUE"` query parametrs, to the scenario with model you want to use submodels for:
```
I'd mention the thing that we need to add to the request (these are query parameters) - and the last comma made this sentence difficult to read to me.